src/feeds/InvPriceFeed.sol

```diff
@@ -0,0 +1,156 @@
+     * @return updatedAt The timestamp when the latest round of Chainlink price feed was updated
+     * @return answeredInRound The round ID in which the answer was computed
+     */
+    function latestRoundData()
+        external
+        view
+        returns (uint80, int256, uint256, uint256, uint80)
+    {
+        return _latestRoundData();
+    }
+
+    /** 
+    @notice Retrieves the latest price for the INV token
+    @return price The latest price for the INV token
+    */
+    function latestAnswer() external view returns (int256) {
+        (, int256 price, , , ) = _latestRoundData();
+        return price;
+    }
+
+    /**
+     * @notice Retrieves number of decimals for the INV price feed
+     * @return decimals The number of decimals for the INV price feed
+     */
+    function decimals() public pure returns (uint8) {
+        return 18;
+    }
+
+    /**
+     * @notice Checks if a given price is out of the boundaries defined in the Chainlink aggregator.
+     * @param price The price to be checked.
+     * @param feed The Chainlink feed to retrieve the boundary information from.
+     * @return bool Returns `true` if the price is out of bounds, otherwise `false`.
+     */
+    function isPriceOutOfBounds(
+        int price,
+        IChainlinkFeed feed
+    ) public view returns (bool) {
+        IAggregator aggregator = IAggregator(feed.aggregator());
+        int192 max = aggregator.maxAnswer();
+        int192 min = aggregator.minAnswer();
+        return (max <= price || min >= price);
+    }
+
+    /**
+     * @notice Fetches the ETH to USD price and the ETH to USDC to get USDC/USD price, adjusts the decimals to match Chainlink oracles.
+     * @dev The function assumes that the `price_oracle` returns the price with 18 decimals, and it adjusts to 8 decimals for compatibility with Chainlink oracles.
+     * @return roundId The round ID of the Chainlink price feed
+     * @return usdcToUsdPrice The latest USDC price in USD computed from the ETH/USD and ETH/USDC feeds
+     * @return startedAt The timestamp when the latest round of Chainlink price feed started
+     * @return updatedAt The timestamp when the latest round of Chainlink price feed was updated
+     * @return answeredInRound The round ID in which the answer was computed
+     */
+    function usdcToUsdFallbackOracle()
+        public
+        view
+        returns (uint80, int256, uint256, uint256, uint80)
+    {
+        int crvEthToUsdc = int(tricrypto.price_oracle(ethK));
+
+        (
+            uint80 roundId,
+            int256 ethToUsdPrice,
+            uint256 startedAt,
+            uint256 updatedAt,
+            uint80 answeredInRound
+        ) = ethToUsd.latestRoundData();
+
+        int256 usdcToUsdPrice = (crvEthToUsdc * 10 ** 8) /
+            ethToUsdPrice /
+            10 ** 10;
+
+        return (roundId, usdcToUsdPrice, startedAt, updatedAt, answeredInRound);
+    }
+
+    function _latestRoundData()
```

Comment:
Should have natspec docs

Reply:
Fixed in [863bbd2](https://github.com/InverseFinance/FiRM/pull/58/commits/863bbd23cbfd871dacf8c690cd21a1285fd941ba)